### PR TITLE
Adds Website docs for cloudflare_static_route

### PR DESCRIPTION
**Pull request by cehrig**
My bad, I forgot the website docs for static routes in the other PR. Here they are :-D

### CODE REVIEW

File: website/docs/r/static_route.html.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+* `prefix` - (Required) Your network prefix using CIDR notation.
+* `nexthop` - (Required) The nexthop IP address where traffic will be routed to.
+* `priority` - (Required) The priority for the static route.
+* `weight` - (Optional) The optional weight for ECMP routes.
+* `colo_names` - (Optional) Optional list of Cloudflare colocation names for this static route.
+* `colo_regions` - (Optional) Optional list of Cloudflare colocation regions for this static route.
+
+## Import
+
+An existing Magic Firewall Ruleset can be imported using the account ID and ruleset ID
```
Comment (jacobbednarz) [inline website/docs/r/static_route.html.markdown:48]:
```suggestion
An existing static route can be imported using the account ID and static route ID
```

**Review by jacobbednarz — COMMENTED** [review]
ssoooo close 😁 just got a small copy/paste to update

File: website/docs/r/static_route.html.markdown (hunk 1 above)
Comment (cehrig) [reply website/docs/r/static_route.html.markdown:48]: Damn, Eagle Eye. Thank You!

**Comment by jacobbednarz** [comment]
👏

**Pull request merged**